draft-ietf-tls-dtls13.md

```diff
@@ -750,6 +765,58 @@ because an attacker will have difficulty forging a datagram that will
 not be rejected by the transport layer.
 
 
+### AEAD Limits
+
+Section 5.5 of TLS {{!TLS13}} defines limits on the number of records that can
+be protected using the same keys. These limits are specific to an AEAD
+algorithm, and apply equally to DTLS. Implementations SHOULD NOT protect more
+records than allowed by the limit specified for the negotiated AEAD.
+
+{{!TLS13}} does not specify a limit for AEAD_AES_128_CCM, but the analysis in
+{{ccm-bounds}} shows that a limit of 2^23 packets can be used to obtain the
+same confidentiality protection as the limits specified in TLS.
+
+The usage limits defined in TLS 1.3 exist for protection against attacks
+on confidentiality and apply to successful applications of AEAD protection. The
+integrity protections in authenticated encryption also depend on limiting the
+number of attempts to forge packets. TLS achieves this by closing connections
+after any record fails an authentication check. In comparison, DTLS ignores any
+packet that cannot be authenticated, allowing multiple forgery attempts.
+
+Implementations MUST count the number of received packets that fail
+authentication. If the number of packets that fail authentication with a single
+key exceeds a limit that is specific to the AEAD in use, an implementation MUST
+immediately close the connection. Implementations SHOULD initiate a key update
+with update_requested before reaching this limit. Once a key update has been
+initiated, the previous keys can be dropped when the limit is reached rather
+than closing the connection. Applying a limit reduces the probability that an
+attacker is able to successfully forge a packet; see {{AEBounds}} and
+{{ROBUST}}.
+
+For AEAD_AES_128_GCM, AEAD_AES_256_GCM, and AEAD_CHACHA20_POLY1305, the limit
+on the number of records that fail authentication is 2^36. Note that the
```

Comment:
"... the number of records ... _is_ 2^36." is quite strong wording, and in particular it's not clear where this number comes from and whether it's taken from one of the papers, a universal, or our own recommendation. I'd suggestt to either name the source here, or add a half sentence like "this specification recommends ..."

Reply:
I'm using a SHOULD instead of MUST for the limit now, so I don't feel like the definitive statement needs to be softened any.

As far as it goes, if you feel like you can drop >60 billion bogus records and keep going, then you are welcome to do so.